src/reynir_correct/errtokenizer.py

```diff
@@ -1885,7 +1889,7 @@ def get_details(
         """Return short and detailed descriptions for the error category plus a link to grammar references where possible"""
         # text is the short version, about the category and the error.
         # details is the long version with references.
-        standref, cat, det = RitmyndirDetails.DICT[code]
+        standref, cat, det = RitmyndirDetails.DICT.get(code, ("", "", ""))
```

Comment:
Is there an early escape out of the function if the code is not found in RitmyndirDetails?